Edits for "conventions" and impersonal form (#628)
cf-convention/cf-conventions: ch01.adoc

The purpose in restricting how the metadata is represented is to make it practical to write software that allows a machine to parse that metadata and to automatically associate each data value with its location in time and space.
It is equally important that the metadata be easy for human users to write and to understand.

These conventions are intended for use with climate and forecast data, for atmosphere, surface and ocean, and was designed with model-generated data particularly in mind.

> It is recognised that there are limits to what a set of conventions can practically cover; these conventions are restricted to issues that are considered to be of common and frequent concern in the design of climate and forecast metadata.

Reviewer: A neater way would be, "Recognising that there ... practically cover, these conventions ...".

Reviewer 2: > climate and forecast data, for atmosphere, surface and ocean, and was designed with model-generated data particularly in mind.

It seems to me that the scope of CF has broadened wuite a bit since it started -- is this statement still true?

Also -- I've always been a bit confused as to what "climate and forecast" means anyway ;-)

I"m not sure I can think of a better way to define the scope, however, but maybe something about:

conventions for oceanographic and meteorological data? 

(though we're expanding into hydrologic and ??? as we go...)

Author: This is a topic that would require a broader discussion. Perhaps open a separate issue on this?

Reviewer: I agree that this is a different issue from the present one, but worth considering elsewhere.

> It is recognised that there are limits to what a set of conventions can practically cover; these conventions are restricted to issues that are considered to be of common and frequent concern in the design of climate and forecast metadata.

Author: ```suggestion
Recognising that there are limits to what a set of conventions can practically cover, these conventions are restricted to issues that are considered to be of common and frequent concern in the design of climate and forecast metadata.
```

The main purpose therefore, is to propose a clear, adequate and flexible definition of the metadata needed for climate and forecast data.

> Although these conventions are specifically targeting the netCDF format, most of the ideas are of wider application.

Reviewer: "Although these are conventions for netCDF specifically, most of the ideas ..."

> Although these conventions are specifically targeting the netCDF format, most of the ideas are of wider application.

Author: ```suggestion
Although these are conventions for netCDF specifically, most of the ideas are of wider application.
```

The metadata objects could be contained in file formats other than netCDF.
Conversion of the metadata between files of different formats will be facilitated if conventions for all formats are based on similar ideas.

> These conventions are designed to be backward compatible with the COARDS conventions <<COARDS>>, which implies that a conforming COARDS dataset also conforms to the CF conventions.

Reviewer: "<<COARDS>>; this means that". I would say it's a definition rather than an implication.

Reviewer 2: I agree -- it's more than in implication. -- is it still fully backward compatible now? or only mostly?

If it's fully backward compatible then we can simply say that.

These conventions are  backward compatible with the COARDS conventions <<COARDS>>;  a conforming COARDS dataset also conforms to the CF conventions.

If mosty, we can say that, too.

These conventions are  designed to be backward compatible with the COARDS conventions <<COARDS>>;  most conforming COARDS datasets also conforms to the CF conventions.

Reviewer: I believe that COARDS-compliant datasets are also compliant with the latest CF version, although there are some COARDS conventions that CF recommends against.

> These conventions are designed to be backward compatible with the COARDS conventions <<COARDS>>, which implies that a conforming COARDS dataset also conforms to the CF conventions.

Author: ```suggestion
These conventions are designed to be backward compatible with the COARDS conventions <<COARDS>>, meaning that a conforming COARDS dataset also conforms to the CF conventions.
```

Thus new applications that implement the CF conventions will be able to process COARDS datasets.

> These conventions also strive to maximize conformance to the COARDS conventions, that is, wherever the COARDS metadata conventions provide an adequate description their use is included here.

Reviewer: I think that only animate agents can strive, and the conventions have not yet got a life of their own. I suggest we replace this with a simpler sentence which has the same intention. "These conventions uphold the COARDS metadata conventions wherever the latter provide an adequate description."

> These conventions also strive to maximize conformance to the COARDS conventions, that is, wherever the COARDS metadata conventions provide an adequate description their use is included here.

Author: ```suggestion
These conventions uphold the COARDS metadata conventions wherever the latter provide an adequate description.
```

[[overview, Section 1.4, "Overview"]]
=== Overview

No variable or dimension names are standardized by these conventions.

> Instead, the lead of the <<NUG>> is followed and only the names of attributes and some of the values taken by those attributes are standardized.

Reviewer: "Instead, following the lead of the <<NUG>>, only the names of attributes ..."

> Instead, the lead of the <<NUG>> is followed and only the names of attributes and some of the values taken by those attributes are standardized.

Author: ```suggestion
Instead, following the lead of the <<NUG>>, only the names of attributes and some of the values taken by those attributes are standardised.
```

Four types of coordinates receive special treatment by these conventions: latitude, longitude, vertical, and time.
Every variable must have associated metadata that allows identification of each such coordinate that is relevant.

> Two independent parts of the conventions allow this to be done.

Reviewer: "Two kinds of conventions allow this to be done."

> Two independent parts of the conventions allow this to be done.

Author: ```suggestion
Two kinds of conventions allow this to be done.
```

These conventions generalize and extend the COARDS conventions <<COARDS>>.
A major design goal has been to maintain __backward compatibility__ with COARDS.
Hence applications written to process datasets that conform to these conventions will also be able to process COARDS conforming datasets.

> The conventions also strive to maximize __conformance__ to the COARDS conventions so that datasets that only require the metadata that was available under COARDS will still be able to be processed by COARDS conforming applications.

Reviewer: "As far as possible, these conventions maximize ...".

> The conventions also strive to maximize __conformance__ to the COARDS conventions so that datasets that only require the metadata that was available under COARDS will still be able to be processed by COARDS conforming applications.

Author: ```suggestion
As far as possible, these conventions maximize __conformance__ to the COARDS conventions so that datasets that only require the metadata that was available under COARDS will still be able to be processed by COARDS conforming applications.
```
